addons/populate/functions/fnc_spawnBuildingUnit.sqf

```diff
@@ -0,0 +1,95 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Spawns a unit at the passed position
+
+    Parameters:
+        0: _spawnPositions <ARRAY>
+        1: _numberOfUnitsToSpawn <NUMBER>
+        2: _numberOfPositionsToOccupy <NUMBER>
+        3: _side <SIDE>
+        4: _module <MODULE>
+        5: _unitPoolArray <ARRAY>
+        6: _statics <ARRAY>
+
+    Return value:
+        Nothing
+*/
+
```

Comment:
I would rework this to separate the spawning of units into statics, positions, and patrols